src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs

```diff
@@ -131,11 +131,15 @@ private string TryGetText(SyntaxToken token, SemanticModel semanticModel, Docume
             return string.Empty;
         }
 
-        private bool TryGetTextForSpecialCharacters(SyntaxToken token, out string text)
+        private static bool TryGetTextForSpecialCharacters(SyntaxToken token, out string text)
         {
             if (token.IsKind(SyntaxKind.InterpolatedStringStartToken) ||
                 token.IsKind(SyntaxKind.InterpolatedStringEndToken) ||
-                token.IsKind(SyntaxKind.InterpolatedStringTextToken))
+                token.IsKind(SyntaxKind.InterpolatedStringTextToken) ||
+                token.IsKind(SyntaxKind.InterpolatedSingleLineRawStringStartToken) ||
+                token.IsKind(SyntaxKind.InterpolatedSingleLineRawStringEndToken) ||
+                token.IsKind(SyntaxKind.InterpolatedMultiLineRawStringStartToken) ||
+                token.IsKind(SyntaxKind.InterpolatedMultiLineRawStringEndToken))
```

Comment:
I think we'd need another conditional branch for `MultiLineRawStringLiteral` and `SingleLineRawStringLiteral` here as well and co-ordinate the F1 keyword to be used with dotnet/docs team.

```csharp
if (token.IsKind(SyntaxKind.MultiLineRawStringLiteral) || token.IsKind(SyntaxKind.SingleLineRawStringLiteral))
{
    text = "RawStringLiteral_CSharpKeyword"; // or """_CSharpKeyword
    return true;
}
```